### DISCUSSION TITLE
Incorrect Commune-to-Wilaya Relations

### DISCUSSION THREAD

**Discussion by n4ss1m** [kossa/algerian-cities#19] [Q&A] [answered]
Examples:

- **El Meniaa**: Currently listed with 2 communes, but it should have 3. [source](https://fr.wikipedia.org/wiki/Liste_des_communes_de_la_wilaya_d%27El_Meniaa)

- **El M'Ghair**: Currently listed with 3 communes, but it should have 8. [source](https://fr.wikipedia.org/wiki/Liste_des_communes_de_la_wilaya_d%27El_M%27Ghair)

- **Touggourt**

- Wilaya 54, 53, 52, etc. even wilayas from 1 - 48 because communes moved after last update

The communes are present, but their provincial associations are incorrect.

I found that this source has accurate, up-to-date data (with additional Daira details): 

https://github.com/othmanus/algeria-cities (we should consider migrating to this data)

The previous source, https://github.com/bahinapster/Wilaya-Of-Algeria/, has outdated information.

**Comment by kossa** [comment]
Right, the good thing we just need you change [json files here](https://github.com/kossa/algerian-cities/tree/master/database/seeders/json) and small change in [seeder](https://github.com/kossa/algerian-cities/blob/master/database/seeders/WilayaCommuneSeeder.php#L44-L89)

**Comment by n4ss1m** [comment, marked as answer]
fixed in https://github.com/kossa/algerian-cities/pull/30
